jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextHandler.java

```diff
@@ -1185,8 +1185,9 @@ protected void doStop() throws Exception
     }
 
     @Override
-    protected ServletContextRequest wrap(Request request, String pathInContext)
+    protected ServletContextRequest wrap(Request request)
     {
+        String pathInContext = getContext().pathInContext(request.getHttpURI().getCanonicalPath());
```

Comment:
Use `Request.getPathInContext(request)` instead?

Reply:
Can't yet, as request is not yet wrapped. We are in process of doing that. I'll write that in a comment